### ISSUE TITLE
Can we please use the same repository for Symfony set of questions ?

### ISSUE THREAD

**Issue by mickaelandrieu** [ProPheT777/certificationy-data#23]
Hi @ProPheT777,

can we discuss about a merge of our current decks:
* I have questions you don't have;
* You have questions I don't have;
* We have the same objectives

We could take time to make a complete review: what do you think ? I plan to add almost 350 questions in the 3 next months (I *may* train myself for the certification, right ? ;) )

**Comment by ProPheT777** [comment]
As I said, i'm in favor of union our both questions deck and i'm also interested to implement the flash card  learning system on web platform.

Are you available on IM (like gitter, skype) ? I will be more simple to discuss about that.

**Comment by mickaelandrieu** [comment]
Yes, can we do that tomorrow, 8 pm ? I'm a little bit busy for now but very motivated to discuss on this with you asap :)

**Comment by ProPheT777** [comment]
Unfortunately Friday night means :beers: :smile: But 24, 24, 26 aug, i'm available from 7 pm, are you ?

**Comment by jameshalsall** [comment]
@mickaelandrieu @ProPheT777 did this happen?
